data/docs/rules-v2.md

```diff
@@ -0,0 +1,204 @@
+title: Comment jouer ?
+
+Dans le Corporate Game, vous incarnez une pointure dans l’univers de la finance dans un futur où le capitalisme a fini par dissoudre les pays. Vous jouez à Detroit, la première ville libérée du contrôle des États. Bienvenue dans le Disneyland des traders !
+
+Ces règles vont détailler toutes les actions que vous avez la possibilité de faire à chaque tour. Le jeu se joue en 7 tours de 24h. À 13h chaque jour, les actions de chaque joueur seront résolues en même temps et un nouveau tour pourra commencer. La personnalité la plus influente de Detroit sera sacrée à la fin du 7<sup>eme</sup> tour, les autres sombreront dans l’oubli.
+
+[TOC]
+
+## Le classement corporatiste
+> Le classement de fin d’année c’est un peu comme une course. Tu peux avoir été devant tout le long, si le dernier te passe devant juste avant la ligne, c’est sa sale tronche qui sera dans le journal. 
+
+Votre objectif est de devenir la personnalité la plus respectée de Detroit à la fin de l’année. Le truc important à retenir, c’est qu’il n’y a pas de victoire en équipe. Il y aura celui qui a réussi et ceux qui ont échoué. Et commencez pas à parler de chance, personne n’aime les gens qui se plaignent !
+ 
+> Pourquoi je ne mise que sur 10 corpos parmi des centaines ? Tu m’as bien regardé ? J’ai l’air d’un hippie qui laisse traîner son fric dans une boite réglo ?
+
+Votre champ de bataille, c’est 10 énormes entreprises cotées à Détroit. On les appelle des corporations, ou **corpos**. Chaque corpo commence avec un nombre d’**actifs** qui va évoluer au cours de la partie, au départ les corpos commencent avec entre 8 et 13 actifs.
+ 
+Les règles de l’économie sont simples :
+
+* Vous commencez la partie avec 2 000 000 ny. Ou 2 000 kilo-NewYen (kny).
+* Acheter une **part** coûte `actifs × 100 kny` (sauf si la corpo est première : `actifs × 125 kny` à la place)
+* Vendre une part rapporte immédiatement `actifs × 100 kny` (sauf si la corpo est première : `actifs × 125 kny` à la place). Cet argent est utilisable pour le tour en cours.
```

Comment:
Mouais... enfin bon, si au dernier moment t'arrive à savoir comment un adversaire va jouer son tour, t'auras probablement des trucs à changer, revente de part ou pas.

You guys know where I stand.